dask/diagnostics/tests/test_profiler.py

```diff
@@ -21,6 +22,8 @@
        'd': (mul, 'a', 'b'),
        'e': (mul, 'c', 'd')}
 
+dsk2 = {'a': 1, 'b': 2, 'c': (lambda a, b: sleep(1) or (a + b), 'a', 'b')}
```

Comment:
Trying to ensure the task is >> `dt`. There's some overhead to collecting resource information, so `dt` can only be so small. Could probably drop to 0.5 s or something if needed.